prac_06/languages.py

```diff
@@ -4,3 +4,19 @@
 actual =
 """
 
+from programming_language import ProgrammingLanguage
```

Comment:
Everything in languages seems great. The only thing I spotted that might have a better way is adding prac_06.programming_language instead of just doing from programming_language. See car class for example.